Add openpype_mongo command flag for testing.

tokejepsen opened the pull request:
## Changelog Description
Instead of changing the environment, this command flag allows for changing the database.

## Testing notes:
1. Run test suite with command flag `--openpype_mongo [URI]`.

iLLiCiTiT commented on openpype/cli.py:
```diff
@@ -290,11 +290,15 @@ def run(script):
               "--setup_only",
               help="Only create dbs, do not run tests",
               default=None)
+@click.option("--openpype_mongo",
```
I would say that `mongo_url` would be better name for cli.

kalisp reviewed (approved):
Had to revert TEST_OPENPYPE_MONGO as it is used in existing .zip files.
Fixed `openpype_mongo` fixture to be actually used.
Renamed `openpype_mongo` to `mongo_url` according to Kuba's comment.

tokejepsen commented:
> Had to revert TEST_OPENPYPE_MONGO as it is used in existing .zip files. Fixed `openpype_mongo` fixture to be actually used. Renamed `openpype_mongo` to `mongo_url` according to Kuba's comment.
> 
> Please revisit/test on your side and I could merge this.

Looks good to me. Merge at will!
